open-budget/app/scripts/models.coffee

```diff
@@ -426,6 +430,22 @@ class ReadyAggregator
             @ready = true
             pageModel.trigger(@event)
 
+# window.onresize can only hold 1 callback, the ResizeNotifier will serve as
+# an initiator for onresize events
+class ResizeNotifier extends Backbone.Model
```

Comment:
doesn't really need to be a Backbone.Model... a simple class can suffice I think